src/hooks/useSSO.ts

```diff
@@ -0,0 +1,24 @@
+import { useCallback } from "react";
+import firebase from "firebase/app";
+
+import { SSO_PROVIDER_ID } from "secrets";
```

Comment:
[NB] May be considered OOS of this PR, but instead of needing to configure this at the environment secrets level, we should configure it on the sovereign `venue` in firebase (and eventually it would be on the `world`)

`LoginForm` already seems to have `venue` in scope anyway, so would be easy to pass the relevant details through here. That way we would also be able to easily configure different SSO methods for different venues (and would take less env setup/code changes to do so, etc)

Reply:
> LoginForm already seems to have venue in scope anyway, so would be easy to pass the relevant details through here. That way we would also be able to easily configure different SSO methods for different venues (and would take less env setup/code changes to do so, etc)

Done in https://github.com/sparkletown/sparkle/pull/1546/commits/cd6ed2a394579a1a482db8a751630924672ad528

Reply:
> [NB] May be considered OOS of this PR, but instead of needing to configure this at the environment secrets level, we should configure it on the sovereign venue in firebase (and eventually it would be on the world)

👍 OOS

Reply:
As per our processes, if you're calling something OOS then you need to make a backlog issue for it + link that from the thread so that it's not lost. Please follow our proccesses.

Reply:
Looking at the code changes it looks like you did this anyway? So no clue why you're calling it OOS?